build_projects/shared-build-targets-utils/Utils/SharedFrameworkPublisher.cs

```diff
@@ -215,16 +214,24 @@ private void GenerateRuntimeGraph(DotNetCli dotnetCli, string destinationDeps)
             }
             if (!string.IsNullOrEmpty(runtimeGraphGeneratorRuntime))
             {
-                var runtimeGraphGeneratorName = "RuntimeGraphGenerator";
-                var runtimeGraphGeneratorProject = Path.Combine(Dirs.RepoRoot, "setuptools", "independent", runtimeGraphGeneratorName);
-                var runtimeGraphGeneratorOutput = Path.Combine(Dirs.Output, "setuptools", "independent", runtimeGraphGeneratorName);
+                var depsProcessorName = "DepsProcessor";
```

Comment:
Is there a specific reason we renamed this? This is a depsprocessor but of a special kind - it generated RID fallback graph.